package.json

```diff
@@ -23,28 +23,28 @@
     "@apollo/client": "^3.7.9",
     "@contentstack/cli-command": "^1.2.19",
     "@contentstack/cli-launch": "^1.0.18",
-    "@contentstack/cli-utilities": "^1.7.1",
-    "adm-zip": "^0.5.10",
+    "@contentstack/cli-utilities": "^1.7.0",
+    "adm-zip": "^0.5.15",
     "chalk": "^4.1.2",
     "lodash": "^4.17.21",
-    "open": "^9.1.0",
+    "open": "^10.0.0",
```

Comment:
@sunil-lakshman , can you check whether we're using open library in app-cli or not ? If not then please remove it.

Reply:
@aman19K The open library has been removed and not used in this package.